pyphare/pyphare/pharein/simulation.py

```diff
@@ -313,6 +360,39 @@ def check_refinement(**kwargs):
 
 
 
+def check_nesting_buffer(**kwargs):
+    nesting_buffer = kwargs.get('nesting_buffer', 0)
+
+    if nesting_buffer < 0:
+        raise ValueError(f"Error: nesting_buffer({nesting_buffer}) cannot be negative")
+
+    smallest_patch_size = kwargs["smallest_patch_size"]
+    largest_patch_size = kwargs["largest_patch_size"]
+
+    if nesting_buffer > smallest_patch_size / 2:
```

Comment:
since nesting_buffer is expressed in terms of coarse cells, I am not sure it makes sense to compare that number to a the patch_size which is expressed in number of *refined* cells. We'd need to include the refinement ratio in there